compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -2377,10 +2393,15 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
     return infer_transpose_conv(node);
   }
 
-  loco::NodeShape visit(const luci::CircleUnpack *node) final { return infer_unpack(node); }
+  loco::NodeShape visit(const luci::CircleUnidirectionalSequenceLSTM *node) final
+  {
+    return infer_unidirectionalsequencelstm(node);
+  }
 
   loco::NodeShape visit(const luci::CircleUnique *node) final { return infer_unique(node); }
 
+  loco::NodeShape visit(const luci::CircleUnpack *node) final { return infer_unpack(node); }
```

Comment:
I think this change be in separate PR :)

Reply:
Okay. When I add a this, I found this is not alphabetic order. so I just thought revise this, but this is not a part of adding new OP. so, I will rollback it.